src/Dictionaries/HashedDictionary.h

```diff
@@ -27,13 +27,20 @@ namespace DB
 struct HashedDictionaryStorageConfiguration
 {
     const bool preallocate;
+    const UInt64 shards;
+    const UInt64 shard_load_queue_backlog;
```

Comment:
Documentation need to be updated, now it is `per_shard_load_queue_backlog`.

Reply:
Looks like you are looking at the old version of the patch:

```
$ gg shard_load_queue_backlog docs
docs/en/sql-reference/dictionaries/external-dictionaries/external-dicts-dict-layout.md:    <shard_load_queue_backlog>10000</shard_load_queue_backlog>
docs/en/sql-reference/dictionaries/external-dictionaries/external-dicts-dict-layout.md:    <!-- <shard_load_queue_backlog>10000</shard_load_queue_backlog> -->
```

P.S. I did rename with `sed`, so everything should be renamed.